terraform/install.sh

```diff
@@ -37,6 +42,7 @@ promptForBillingAccount() {
     log ""
     log "To list active billing accounts, run:"
     log "gcloud beta billing accounts list --filter open=true"
+    python3 telemetry.py $SESSION "No project" no-active-billing $VERSION
```

Comment:
In this instance, no project has been created yet, since the project is only created after an active billing account is created. Instead of using "No project", I could alternatively use a constant.